src/hooks/useProfileStatus.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from "react";
+
+import { useUser } from "hooks/useUser";
+import { UserStatus } from "types/User";
+import { updateUserOnlineStatus } from "api/profile";
+
+export const useProfileStatus = () => {
+  const { userWithId, profile } = useUser();
```

Comment:
you can just import UserId from `useUser`

Reply:
https://github.com/sparkletown/sparkle/pull/1384/commits/eb9038d3849d6a10733530fa9fa7cc100e6e6f95